R/bambu.R

```diff
@@ -145,25 +147,39 @@ bambu <- function(reads = NULL, rcFile = NULL, rcOutDir = NULL,
             rm.readClassSe <- TRUE # remove temporary read class files 
         }
         readClassList <- bambu.processReads(reads, annotations, 
-            genomeSequence = genomeSequence, readClass.outputDir = rcOutDir, 
-            yieldSize, bpParameters, stranded, verbose, isoreParameters)
-    } else {readClassList <- rcFile}
+            genomeSequence = genomeSequence, 
+            readClass.outputDir = rcOutDir, yieldSize, 
+            bpParameters, stranded, verbose,
+            isoreParameters, trackReads = trackReads)
+    } else { readClassList <- rcFile}
```

Comment:
This had some issues in extendAnnotations. Made a minor change and made it so that users can still input a list of rcFile. 
Resolved in b001be6f5b2249955382f849bce8cdd607f7f0fc